lib/presentation/widget/upload_file/upload_file_viewmodel.dart

```diff
@@ -48,23 +48,31 @@ import 'package:linshare_flutter_app/presentation/widget/destination_picker/dest
 import 'package:linshare_flutter_app/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfling_arguments.dart';
 import 'package:linshare_flutter_app/presentation/widget/upload_file/upload_file_arguments.dart';
 import 'package:linshare_flutter_app/presentation/util/extensions/media_type_extension.dart';
+import 'package:permission_handler/permission_handler.dart';
 import 'package:redux/redux.dart';
 import 'package:redux_thunk/redux_thunk.dart';
 import 'package:rxdart/rxdart.dart';
 
 import 'upload_destination_type.dart';
 
+enum ContactSuggestionSource {
+  all, linShareContact, deviceContact
```

Comment:
Can you use enum syntaxe like "ALL, LINSHARE_CONTACT ..." pls ?

Reply:
oops, bad habit for me

Reply:
done